File: 04 - ORIENTAÇÃO A OBJETOS/nodeJS - JavaScript_/Exercicio 6/Com Pacotes/Aluno.js
var Pessoa = require('./Pessoa')

module.exports = class Aluno extends Pessoa{
    constructor(nome, cpf, nota){
        super(nome, cpf)
        this.nota = nota
    }

    mostraAtributos(){
        return `Aluno: ${this.nome}, CPF: ${this.cpf}, Nota: ${this.nota}`
    }
}

File: 04 - ORIENTAÇÃO A OBJETOS/Java/Exercicio 6/Aluno.java

public class Aluno extends Pessoa {
    double nota;
    
    Aluno(String nome, String cpf, double nota){
        super(nome, cpf);
        this.nota = nota;
    }
    
    void mostraAtributos(){
        System.out.println("Aluno: "+super.nome+". CPF: "
                +super.cpf+". Nota: "+this.nota);
    }
}
